packages/clerk-js/src/ui/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -91,14 +95,70 @@ export const ImpersonationFab = () => {
   const session = useCoreSession();
   const { t } = useLocalizations();
   const { parsedInternalTheme } = useAppearance();
+  const containerRef = useRef<HTMLDivElement>(null);
   const actor = session?.actor;
   const isImpersonating = !!actor;
 
   //essentials for calcs
-  const top = '109px';
-  const right = '23px';
   const eyeWidth = parsedInternalTheme.sizes.$16;
   const eyeHeight = eyeWidth;
+  const topProperty = '--cl-impersonation-fab-top';
+  const rightProperty = '--cl-impersonation-fab-right';
+  const defaultTop = 109;
```

Comment:
❓ Where do these values come from?

Reply:
These properties are being set when dragging the Fab